modules/ITK/cli/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+if(MSVC11) #i.e. Visual Studio 2012
+	# Fix for VS2012 that has _VARIADIC_MAX set to 10. Don't set too high because it increases compiler memory usage / compile-time.
+	add_definitions( -D_VARIADIC_MAX=10 )
+	# Fix for another VS2012 problem: not all TR1 options are automatically detected, therefore we force them here.
+	add_definitions( -D BOOST_HAS_TR1 )
+	add_definitions( -D BOOST_NO_0X_HDR_INITIALIZER_LIST )
+endif()
+
+#the idea here is to have an X.cxx and a X.md where X denotes the cli-command name.
+set( _cli_files
+	statismo-build-shape-model
+)
+
+#search for pandoc to generate man pages or chm help files
+find_program(PANDOC NAMES pandoc)
```

Comment:
this find_program should be moved in the main CMakeLists.txt else you would copy paste this code for every module